### DISCUSSION TITLE
Consolidating the name AutomaticallyUnload and UnusedMode in SceneLoadData and SceneUnloadData respectfully.

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#201] [Documentation]
Hey FGG,

As I am going through the documentation for SceneManagement, I am creating a page about the ability for the server to keep a scene loaded when no clients are observing that scene anymore. 

I was hoping we could use this naming convention in the SceneLoadData and SceneUnloadData options instead of currently having two different naming conventions that IMO get confusing.

### **Suggested**
Calling the page "Scene Caching"  

**SceneLoadData** 
Current: AutomaticallyUnload - Options: true, false.
Proposed: CacheScenes - Options: true, false

**SceneUnloadData**
Current: Mode - Options: ServerUnloadMode.UnloadUnused, ServerUnloadMode.KeepUnused.
Proposed: CacheScenes - Options: true, false

### **Alternative**

Alternatively we could call the page "Unused Scenes"
**SceneLoadData** 
Current: AutomaticallyUnload - Options: true, false.
Proposed: UnusedSceneMode - Options: UnusedSceneMode.UnloadUnused, UnloadSceneMode.KeepUnused

**SceneUnloadData**
Current: Mode - Options: ServerUnloadMode.UnloadUnused, ServerUnloadMode.KeepUnused.
Proposed: UnusedSceneMode - Options: UnusedSceneMode.UnloadUnused, UnloadSceneMode.KeepUnused

### Conclusion

In the end I am just hoping we can unify the naming for simplicity no matter what is decided on what the name should be.

**Comment by FirstGearGames** [comment]
As we discussed in open development, this isn't going to work.
Making these changes would break 100% of user projects and that's not something I want to do for renames, even if they do make more sense.
